src/main/java/org/isf/accounting/service/AccountingIoOperations.java

```diff
@@ -329,24 +342,55 @@ public int compare(Bill o1, Bill o2) {
 	 * @param patient
 	 * @return
 	 * @throws OHServiceException
+	 * @deprecated use {@link #getPaymentsBetweenDatesWherePatient(GregorianCalendar, GregorianCalendar, Patient)}
 	 */
+	@Deprecated
```

Comment:
(TIL: from Java 9 you can add since and forRemoval attributes https://www.baeldung.com/java-deprecated#optional-attributes )